### ISSUE TITLE
[BUG] Enable to get the WPF getting started working

### ISSUE THREAD

**Issue by Ehrode** [reactiveui/ReactiveUI#2742]
Hello

**Describe the bug**
After downloading the getting started project from git :
https://github.com/reactiveui/ReactiveUI.Samples/tree/main/wpf/getting-started

I'm not able to get the xaml part working.
I get the following error on the main window : The name "ReactiveWindow" does not exist in the namespace "http://reactiveui.net"
See attached screenshot

I restored NuGet package.
I rebuilt the solution.
All the C# code works like charm, only xaml views and user controls get this namespace error.

**Steps To Reproduce**
Clone the repository : https://github.com/reactiveui/ReactiveUI.Samples.git
Open solution : ReactiveUI.Samples/wpf/getting-started/ReactiveDemo.sln
Restore nuget
Rebuild solution
Open MainWindow.Xaml
Notice the xaml error

**Expected behavior**
The project build and run

**Screenshots**
![image](https://user-images.githubusercontent.com/40527875/114948073-9fc41e80-9e4e-11eb-880b-fa6d11732e95.png)
![image](https://user-images.githubusercontent.com/40527875/114948582-912a3700-9e4f-11eb-814f-7829471c0754.png)

**Comment by starknguyen** [comment]
@Ehrode It looks like you have not set the TypeArguments to ViewModel in MainWindow.xaml. Also, in the code behind of MainWindow.xaml, you should also derive MainWindow from ReactiveWindow<TViewModel>

**Comment by Ehrode** [comment]
Hello, thanks for your answer.
I finally managed to solve my issue.
First, sorry for my screenshot, it was missleading. In fact, I removed the x:TypeArgument when I was trying to solve the problem by myself and forgot to put it back when I made the screenshot.
The solution was to add a reference to System.Xaml that was missing from the project source and for a reason I don't get move the VIewModels away from the View and make a reference to theme like this :
    xmlns:vm="clr-namespace:ReactiveDemo.ViewModels"
    x:TypeArguments="vm:AppViewModel"

If I keep the ViewModels next to their view so the code looks like :
     xmlns:local="clr-namespace:ReactiveDemo"
    x:TypeArguments="local:AppViewModel"

It doesn't works....
But not a big deal since I would always put my viewmodels inside a separate folder anyway.
